Src/coffeescript/cql-execution/src/elm/aggregate.coffee

```diff
@@ -100,6 +125,9 @@ module.exports.MaxFunctionRef = class MaxFunctionRef extends FunctionRef
   exec: (ctx) ->
     @func.exec(ctx)
 
+doAverage = (list) ->
```

Comment:
I don't think this function is used, but if it were, I think there is a bug.  `list` is passed in, but `filtered` is used in the body.  I think we can delete this function, but if not, we should fix the bug.